src/v2/view-builder/ViewFactory.js

```diff
@@ -87,8 +88,8 @@ const VIEWS_MAPPING = {
     email: RequiredFactorEmailView,
     password: RequiredFactorPasswordView,
     webauthn: RequiredFactorWebauthnView,
-    phone: null,
     'security_question': ChallengeAuthenticatorSecurityQuestion,
+    phone: ChallengeAuthenticatorPhoneView,
```

Comment:
@haishengwu-okta was wondering what line #84 in this file is for..

Reply:
you can remove it. it's placeholder for phone enrollment which uses different form name (`select-authenticator-enroll-data`)